src/utils/mod.rs

```diff
@@ -2,3 +2,11 @@ pub mod check;
 pub mod params;
 pub mod path;
 pub mod read;
+
+/// Returns the name of the operator used in the Helm repository.
+pub fn operator_name<T>(name: T) -> String
```

Comment:
Also here.

Reply:
I'd also consider naming it something like `operator_chart_name`.. the current name is quite ambiguous.

Reply:
Changed in df0076f.